README.md

```diff
@@ -14,7 +14,7 @@ Spokestack has all the tools you need to build amazing user experiences for spee
 
 - Automatic Speech Recognition (ASR): We provide multiple ways to hook up either [Spokestack ASR](https://www.spokestack.io/docs/concepts/asr) or [Google Cloud Speech](https://github.com/googleapis/nodejs-speech) to your node/express server, including asr functions for one-off ASR requests and websocket server integrations for ASR streaming. Or, use the ASR services directly for more advanced integrations.
 - Speech-to-Text: Through the use of our [GraphQL](https://graphql.org) API (see below), Spokestack offers multiple ways to generate voice audio from text. Send raw text, [speech markdown](https://www.speechmarkdown.org), or [SSML](https://en.wikipedia.org/wiki/Speech_Synthesis_Markup_Language) and get back a URL for audio to play in the browser.
-- Wake word and Keyword: Wake word and keyword processing are supported through the use of our speech pipeline (see [startPipeline](#startPipeline)). One of the most powerful features we provide is the ability to define and train custom wake word and keyword models directly on [spokestack.io](https://www.spokestack.io/account). When training is finished, we host the model files for you on a CDN. Pass the CDN URLs to `startPipeline()` and the Speech Pipeline will start listening. The pipeline uses a web worker in the browser to keep all of the speech processing off the main thread so your UI never gets blocked.
+- Wake word and Keyword: Wake word and keyword processing are supported through the use of our speech pipeline (see [startPipeline](#startPipeline)). One of the most powerful features we provide is the ability to define and train custom wake word and keyword models directly on [spokestack.io](https://www.spokestack.io/account). When training is finished, we host the model files for you on a CDN. Pass the CDN URLs to `startPipeline()` and the Speech Pipeline will start listening These same models can be used in [spokestack-python](https://github.com/spokestack/spokestack-python), [spokestack-android](https://github.com/spokestack/spokestack-android), [spokestack-ios](https://github.com/spokestack/spokestack-ios), and [react-native-spokestack](https://github.com/spokestack/react-native-spokestack). The pipeline uses a web worker in the browser to keep all of the speech processing off the main thread so your UI never gets blocked. **NOTE: The speech pipeline (specifically tensorflow's webgl backend) currently only works in Blink browsers (Chrome, Edge, Opera, Vivaldi, Brave, and most Android browsers) as it requires the use of the experimental [OffscreenCanvas API](https://caniuse.com/?search=offscreencanvas). Firefox support is already available via a flag, so we expect it to be available there soon.**
```

Comment:
This isn't wrong, but without the ability to choose a sample rate for the user media device, even the webgl backend will probably be too slow to run the models in near-real time.

Reply:
Yes, but can't you choose a sample rate in the same browsers that support OffscreenCanvas?

Reply:
Oh, you're saying FF may not work when it has OffscreenCanvas.

Reply:
Yeah, that's the issue—even with webgl support, FF won't let you change the sample rate for the media device/stream and, while it will let you choose the sample rate for an `AudioContext`, it won't let you connect a stream to an `AudioContext` with a different sample rate.

Reply:
Nvm, according to other reports, setting sampleRate in FF works. Now I forget where it wasn't working.

Reply:
oh I was looking at audiocontext. Let me check bugs for media device.

Reply:
The error message is `Connecting AudioNodes from AudioContexts with different sample-rate is currently not supported.`, fwiw

Reply:
I found this: https://bugzilla.mozilla.org/show_bug.cgi?id=1388586

Is this the right bug? It's weird that's been open for 4 years and still marked NEW, and it doesn't seem high priority. I wonder if we're doing something wrong.

Reply:
It is...one of the bugs. There are others, and this isn't the only way to fix it. Many have been open for years, and they don't appear to have an interest in fully implementing the Web Audio API (no browser seems to).

Reply:
It was the only open FF bug I could find that had the word `sampleRate`.

Reply:
See new comment.